src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1091,6 +1104,27 @@ private void WriteContentFilesToPreprocess()
                     });
             }
 
+            private void WriteDebugSymbolsFiles()
+            {
+                WriteItems(
+                    _runtimeTarget,
+                    package => package.RuntimeAssemblies,
+                    filter: asset => asset.Properties.ContainsKey(MetadataKeys.Related),
+                    writeMetadata: (package, asset) =>
+                    {
+                        asset.Properties.TryGetValue(MetadataKeys.Related, out string propertyValue);
+                        if (string.Compare(propertyValue, MetadataKeys.Pdb, true) == 0)
```

Comment:
Change this code to accept related = "pdb;xml"

Reply:
https://github.com/NuGet/Home/blob/dev/proposed/2021/NuGetConsumePDBInPackageReference.md

Reply:
@heng-liu clarified this off line. The correct is ".pdb;.xml"